src/greentest/test__threading_holding_lock_while_monkey.py

```diff
@@ -1,8 +0,0 @@
-from gevent import monkey
```

Comment:
My new test__monkey_oldlocks_locked.py provides more coverage than that test provided so I removed it to reduce redundancy.